site/it/community/contribute/code.md

```diff
@@ -0,0 +1,136 @@
+# Contributo al codice TensorFlow
+
+Sia che tu voglia aggiungere una funzione perdita, migliorare la copertura di un test, oppure scrivere una
+RFC per un cambiamento rilevante di progettazione, questa parte della guida del contributore ti sarà d'aiuto
+per cominciare. Grazie per il lavoro e l'interesse nel miglioramento di TensorFlow.
+
+## Prima di cominciare
+
+Prima di contribuire al codice sorgente di un progetto TensorFlow, cortesemente consulta il file `CONTRIBUTING.md` nel repository GitHub del progetto. (Per esempio, vedi
+[file CONTRIBUTING.md nel repository centrale di TensorFlow](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md).) A tutti i contributori è richiesto di firmare un [Accordo di Licenza di Contribuzione](https://cla.developers.google.com/clas) (CLA).
+
+Per evitare attività duplicate, prima di cominciare a lavorare su una funzionalità non banale, cortesemente consulta [RFCs correnti](https://github.com/tensorflow/community/tree/master/rfcs) e contatta gli sviluppatori nei forum TensorFlow. Talvolta siamo selettivi quando dobbiamo decidere se aggiungere una funzionalità, ed il modo migliore di contribuire ed aiutare il progetto è lavorare su problemi conosciuti. 
+
+## Temi per il nuovo contributore
```

Comment:
`Temi` could be untraslated `Issues` cause is referring to Github Issue that is not translated